src/checkout-buttons/strategies/masterpass/masterpass-button-strategy.ts

```diff
@@ -142,4 +139,11 @@ export default class MasterpassButtonStrategy implements CheckoutButtonStrategy
 
         this._masterpassClient.checkout(this._createMasterpassPayload());
     }
+
+    private formatLocale( localeLanguage: string): string {
+        const locale = localeLanguage.replace(/[-_]/g, '_');
+        const formatedLocale = locale.includes('_') ? `${locale}` : `${locale}_${locale}`;
```

Comment:
So if the locale is `en`, you want to return `en_en`? I think `en_en` is not a valid locale.